### DISCUSSION TITLE
Abilty to calculate SMA

### DISCUSSION THREAD

**Discussion by gonz177** [mito-ds/mito#600] [Feature Requests]
Hi guys, as someone comming from an "excel world", i love how mito makes it easier to work on data! But i am missing the flexibility of xls when it comes to formulas. 

As an analyst, i would like to calculate a Simple Moving Average over a given number of periods. E.g. given a column "usage" i want to calculate SMA over the usage of the last 10 days and have the value in a new cell.  

Thanks!

**Comment by aarondr77** [comment]
Hey @gonz177 sorry for the slow response. This is now possible as of the most recent release, `mitosheet 0.1.471`. 

You can do this all through spreadsheet formulas following these steps:
1. Create a new column called Simple Moving Average
2. In cell 10 of the Simple Moving Average column, write the formulas =AVG(usage0:usage10). 

The formula will calculate the average of the previous 10 days for each row. 

If you get a chance to give it a try, would love to hear your feedback.
